test/snippet/alignment/matrix/debug_matrix.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alignment/matrix/debug_matrix.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/range/view/to_char.hpp>
+
+int main()
+{
+    using namespace seqan3;
+{
+    debug_stream << "//! [score_matrix::out]" << std::endl;
```

Comment:
Why the indentation, and why not globally defined anyway? 
Can you make this a standalone include snippet, please 👼